adapter/src/grpc.rs

```diff
@@ -51,22 +52,56 @@ impl EngineGRPC for EngineGRPCImpl {
         let adapter = self.adapter.clone();
 
         tokio::spawn(async move {
-            let response = adapter.execute_query_impl(request).await.unwrap();
+            let response = adapter.execute_raw_query_impl(request).await.unwrap();
 
             let response_fb = FlatbuffersEngineProtocol
-                .write_execute_query_response(&response)
+                .write_raw_query_response(&response)
                 .unwrap();
 
-            let response_grpc = ExecuteQueryResponseGRPC {
+            let response_grpc = RawQueryResponseGRPC {
+                flatbuffer: response_fb.collapse_vec(),
+            };
+
+            tx.send(Ok(response_grpc)).await.unwrap();
+        });
+
+        Ok(Response::new(ReceiverStream::new(rx)))
+    }
+
+    async fn execute_transform(
+        &self,
+        request_grpc: Request<TransformRequestGRPC>,
+    ) -> Result<Response<Self::ExecuteTransformStream>, Status> {
+        let span = tracing::span!(tracing::Level::INFO, "execute_transform");
+        let _enter = span.enter();
+
+        let request = FlatbuffersEngineProtocol
+            .read_transform_request(&request_grpc.get_ref().flatbuffer)
+            .unwrap();
+
+        info!(message = "Got request", request = ?request);
+
+        let (tx, rx) = tokio::sync::mpsc::channel(1);
+
+        let adapter = self.adapter.clone();
+
+        tokio::spawn(async move {
+            let response = adapter.execute_transform_impl(request).await.unwrap();
+
+            let response_fb = FlatbuffersEngineProtocol
+                .write_transform_response(&response)
+                .unwrap();
+
+            let response_grpc = TransformResponseGRPC {
                 flatbuffer: response_fb.collapse_vec(),
             };
 
             tx.send(Ok(response_grpc)).await.unwrap();
 
             // let responses = [
-            //     ExecuteQueryResponse::Progress,
-            //     ExecuteQueryResponse::Progress,
-            //     ExecuteQueryResponse::Error,
+            //     TransformResponse::Progress,
+            //     TransformResponse::Progress,
+            //     TransformResponse::Error,
```

Comment:
A file comment: could you please add a comment re commented code?

... or if it isn't needed, please delete it